src/Tickethistory.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+
+type Transaction = {
+  id: number
+  datetime: Date,
+  activity: String,
+  amount: number
+};
+type TransactionList = {
+  ticketTransaction: Array<Transaction>
+};
+
+function Tickethistory() {
```

Comment:
You could use PascalCase to make TicketHistory more readable. Make sure to update the filename too and all the imports.